Configuration/Development/Docker/Settings.yaml

```diff
@@ -0,0 +1,40 @@
+#                                                                        #
+# Example Settings                                                       #
+#                                                                        #
+# This file contains settings for various parts of the application.      #
+# Copy this file to Settings.yaml, and adjust as necessary.              #
+#                                                                        #
+# Please refer to the default settings file(s) or the manuals for        #
+# possible configuration options.                                        #
+#                                                                        #
+
+Neos:
+  Flow:
+    persistence:
+      # For a reference of the possible options, take a look at
+      # https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html
+      backendOptions:
+        charset: 'utf8mb4'
+        host: '%env:DB_NEOS_HOST%'
+        dbname: '%env:DB_NEOS_DATABASE%'       # adjust to your database name
+        user: '%env:DB_NEOS_USER%'             # adjust to your database user
+        password: '%env:DB_NEOS_PASSWORD%'         # adjust to your database password
+
+       # if you want to log executed SQL queries, enable the next 2 lines
+#      doctrine:
+#        sqlLogger: 'Neos\Flow\Persistence\Doctrine\Logging\SqlLogger'
+
+    # If you are running Flow within a Docker environment, you probably need to allow all proxies,
+    # because the container acts as such and the IP is variable.
+#    http:
+#      trustedProxies:
+#        proxies: '*'
+
+  Imagine:
+    driver: Gmagick
+
+
+Sandstorm:
```

Comment:
This does not really belong here

Reply:
my bad, I already used the new docker setup to work 🙈